module: add variables allocate_public_connection, resource_group_id and cold_storage for resource alicloud_click_house_db_cluster

itkillsnone opened the pull request:
resource/alicloud_click_house_db_cluster: update example module for allocating public connection.
resource/alicloud_click_house_db_cluster: update example module for using resource group.
resource/alicloud_click_house_db_cluster: update example module for using cold storage.

itkillsnone committed:
resource/alicloud_click_house_db_cluster: update example module for multi-zone instance.

itkillsnone committed:
Merge branch 'alibabacloud-automation:main' into main

shanye997 commented on examples/complete/main.tf:
```diff
@@ -1,11 +1,14 @@
-variable "region" {
-  default = "cn-beijing"
+terraform {
+  required_version = ">= 0.13"
```
这个创建一个 versions.tf，写到里面

itkillsnone replied:
已修改

itkillsnone committed:
resource/alicloud_click_house_db_cluster: update example module for allocating public connection.
resource/alicloud_click_house_db_cluster: update example module for using resource group.
resource/alicloud_click_house_db_cluster: update example module for using cold storage.
